addons/forum-user-agent/addon.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "Modify version information on forums",
+  "description": "Changes the text \"Flash version\" to \"Flash versions\" when clicking the globe icon on the forums.",
+  "info": [
+    {
+      "text": "This is useful for Scratch Addons developers to see if a bug that was reported on the forums was possibly caused by the extension.",
+      "id": "usefulness"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["editingScreens"]
+    }
+  ],
+  "tags": ["community", "forums"],
+  "versionAdded": "1.40.0",
```

```suggestion
  "versionAdded": "1.39.99",
```

The settings page doesn't consider addons where versionAdded == latestUpdate.version, so let's lie to it...

We can restore this to `1.40.1` before releasing 1.41.0

---

```suggestion
  "versionAdded": "1.0.0",
```

This may break less things

---

If you want to check what was the version when we added this functionality to cs.js, go ahead...

---

> If you want to check what was the version when we added this functionality to cs.js, go ahead...

It's 1.13.0.